Documentation/getting-started.md

```diff
@@ -0,0 +1,82 @@
+# Getting Started with Analytics SDK
```

Comment:
Can we follow the Audience format:
1. Have a title for Adobe Analytics and with description 
The [Analytics](https://developer.adobe.com/client-sdks/documentation/adobe-analytics/) extension represents the Analytics Adobe Experience Platform SDK that is required for sending mobile interaction data from your app to Adobe Analytics.

2. Move the dependence inside the Add Analytics extension to your app.
3. Add Configuring the Analytics extension in the Data Collection UI

Reply:
Emm ... I just followed the steps we use in the Assurance repo. I think it should be better to update it in another PR.

Reply:
yes.